modules/nlu/src/translations/fr.json

```diff
@@ -0,0 +1,82 @@
+    "contextSelectorCreateMissing": "Créer \"{query}\"",
+    "contextSelectorPlaceholder": "Sélectionnez une catégorie"
+  },
+  "slots": {
+    "new": "Créer un paramètre",
+    "save": "Enregistrer",
+    "emptyState": "Aucun paramètre n'est défini",
+    "noSlotsToTag": "La sélection ne peut pas être marquée. Créer dabord un paramètre",
+    "tagSelectionLabel": "Marquer la sélection",
+    "tagSelectionMessage": "Cliquer sur un paramètre ou utiliser les nombres comme racourcis.",
+    "nameLabel": "Nom du paramètre",
+    "namePlaceholder": "Nom içi",
+    "entitiesLabel": "Entités associées",
+    "editTitle": "Modifier un paramètre",
+    "createTitle": "Créer un paramètre pour votre intention",
+    "delete": "Supprimer",
+    "edit": "Modifier",
+    "deleteMessage": "Êtes-vous certain de vouloir supprimer ce paramètre ainsi que toutes les marques dans chaque alternative?"
+  },
+  "entities": {
+    "entities": "Entités",
+    "selectPlaceholder": "Sélectionner des entitiées",
+    "filterPlaceholder": "Filter entities",
+    "deleteMessage": "Êtes-vous certain de vouloir supprimer l'entité \"{entityName}\" ?",
+    "sensitiveLabel": "Contient des données sensibles",
+    "sensitiveTooltip": "Les données sensibles sont remplacées par ** avant d'être enregistré dans la base de données",
+    "matchCaseLabel": "Sensible à la casse",
+    "matchCaseTooltip": "Si votre expression doit être sensible à la casse",
+    "optionsLabel": "Options",
+    "examplesPlaceholder": "Ajouter des exemples qui respectent votre expression (un exemple par ligne)",
+    "matchingSuccess": "Tous les exemples sont valides",
+    "matchingError": "Un ou plusieurs exemples ne sont pas valides",
+    "examplesLabel": "Exemples valides",
+    "patternLabel": "Expression régulière",
+    "patternPlaceholder": "Ajouter une expression régulière valide",
+    "patternInvalid": "Expression régulière invalide",
+    "synonymPlaceholder": "Ajouter des synonymes, séparés par des virgules",
+    "strict": "Stricte",
+    "medium": "Moyen",
+    "loose": "Indulgent",
+    "occurrenceTooltip": "Une variante est une valeur que peut prendre votre entité. Chaque variante peut avoir plusieurs synonymes.",
+    "newOccurrence": "Nouvelle variante",
+    "occurrenceLabel": "Variantes",
+    "occurrencePlaceholder": "Entrer une valeur (ex: Chicago)",
+    "fuzzyLabel": "Options de correspondance",
+    "fuzzyTooltip": "Les options de correspondance permettront de fines erreurs (e.g erreur de frappe) dans des mots de 4 caractères ou plus. Stricte signifie qu'aucune erreur n'est permise.",
+    "nameConflictTitle": "Ce nom est déjà utilisé",
+    "nameConflictMessage": "Une entité portant le même nom existe déjà. Veuillez choisir un autre nom.",
+    "namePlaceholder": "Nom de l'entité",
+    "nameLabel": "Nom"
+  },
+  "renameActionLabel": "Renommer",
+  "duplicateActionLabel": "Dupliquer",
+  "deleteActionLabel": "Supprimer",
+  "createActionLabel": "Créer",
```

Comment:
These 4 should be taken from ui-shared (and be global). Same for nameLabel just before